Sources/StreamChatUI/ChatMessageList/Attachments/Poll/PollCreationVC/PollCreationVC.swift

```diff
@@ -0,0 +1,685 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// The sections for the poll creation view.
+public struct PollCreationSection: RawRepresentable, Equatable {
+    public var rawValue: String
+
+    public init(rawValue: String) {
+        self.rawValue = rawValue
+    }
+
+    /// The section to edit the name of the poll.
+    public static var name = Self(rawValue: "name")
+    /// The section to provide the options of the poll.
+    public static var options = Self(rawValue: "options")
+    /// THe section to enable or disable the poll features.
+    public static var features = Self(rawValue: "features")
+}
+
+/// The sections for the poll creation view.
+public struct PollFeatureType: Equatable {
+    public var rawValue: String
+
+    public init(rawValue: String) {
+        self.rawValue = rawValue
+    }
+
+    public static var multipleVotes = Self(rawValue: "multiple-votes")
+    public static var anonymous = Self(rawValue: "anonymous")
+    public static var suggestions = Self(rawValue: "suggestions")
+    public static var comments = Self(rawValue: "comments")
+
+    public static func == (lhs: PollFeatureType, rhs: PollFeatureType) -> Bool {
+        lhs.rawValue == rhs.rawValue
+    }
```

Comment:
Seems to be not needed

Reply:
Yes correct probably a left over from previous implementations, will remove it 👍

Reply:
Done ✅